### DISCUSSION TITLE
Can i start SMPlayer to play a playlist at a specific time?

### DISCUSSION THREAD

**Discussion by stefan-franz** [smplayer-dev/smplayer#460] [Q&A]
Hello, i work with Linux Mint 20.3 - Cinnamon. 
Can i start the SMPlayer with a predefined Playlist in fullview mode that it plays after that a entire playlist?
If yes - how can i do it?

**Comment by smplayer-dev** [comment]
`smplayer playlist.m3u8 -fullscreen`

**Comment by stefan-franz** [comment]
for normal use i switched off that the SMPlayer starts on loading a playlist. 
What is the command line to: 
- Load SMPlayer
- in Fullscreen
- Load a Playlist 
- then play

My try is not working: (playlist ist loaded and the player starts - but not is playing)
smplayer -fullscreen "/home/stefan/Daten/Nextcloud/Sport/0-Playlists/71f  Yogaworks 1.m3u" -send-action play

**Comment by smplayer-dev** [comment]
Enable the option "start playback after loading a playlist" in preferences -> playlist.
